docs/source/delta-sharing.md

```diff
@@ -0,0 +1,215 @@
+
+```scala
+val tablePath = "<profile-file-path>#<share-name>.<schema-name>.<table-name>"
+
+spark.readStream.format("deltaSharing").load(tablePath)
+```
+
+Delta Sharing Streaming supports the following functionalities in the same way as Delta Streaming: [Limit input rate](delta-streaming.md#limit-input-rate), 
+[Ignore updates and deletes](delta-streaming.md#ignore-updates-and-deletes), [Specify initial position](delta-streaming.md#specify-initial-position)
+
+In addition, `maxVersionsPerRpc` is provided to decide how many versions of files are requested from the server in every Delta Sharing rpc. This is to help
+reduce the per rpc workload and make the Delta sharing streaming job more stable,  
+especially when many new versions have accumulated when the streaming resumes from a checkpoint. The default is 100.
+
+.. note:: Trigger.AvailableNow is not supported in Delta Sharing Streaming. You can use Trigger.Once as a workaround, and at a proper frequency to catch up with the changes in the server.
+
+## Read Advanced <Delta> Features in Delta Sharing
+In order to support advanced <Delta> features in Delta Sharing, "Delta Format Sharing" was introduced since delta-sharing-client 1.0 and delta-sharing-spark 3.1, 
+in which the actions of a shared table are returned in <Delta> format, allowing a <Delta> library to read it.
+
+Please remember to set the spark configurations mentioned in [_](delta-batch.md#sql-support) in order to read shared tables with Deletion Vectors and Column Mapping. 
+
+| Read Table Feature | Available since version |
+| - | - |
+| [Deletion Vectors](delta-deletion-vectors.md) | 3.1.0 | 
+| [Column Mapping](delta-column-mapping.md) | 3.1.0 |
+
+Batch queries can be performed as is, because it can automatically resolve the `responseFormat` based on the table features of the shared table.
+An additional option `responseFormat=delta` needs to be set for cdf and streaming queries when reading shared tables with Deletion Vectors or Column Mapping enabled.
+
+```scala
+import org.apache.spark.sql.SparkSession
+
+val spark = SparkSession
+        .builder()
+        .appName("...")
+        .master("...")
+        .config("spark.sql.extensions", "io.delta.sql.DeltaSparkSessionExtension")
+        .config("spark.sql.catalog.spark_catalog", "org.apache.spark.sql.delta.catalog.DeltaCatalog")
+        .getOrCreate()
+
+val tablePath = "<profile-file-path>#<share-name>.<schema-name>.<table-name>"
+
+// Batch query
+spark.read.format("deltaSharing").load(tablePath)
+
+// CDF query
+spark.read.format("deltaSharing")
+  .option("readChangeFeed", "true")
+  .option("responseFormat", "delta")
+  .option("startingVersion", 1)
+  .load(tablePath)
+
+// Streaming query
+spark.readStream.format("deltaSharing").option("responseFormat", "delta").load(tablePath)
+```
+
+.. include:: /shared/replacements.md
```

Comment:
Do we need a troubleshooting section?

Reply:
There is a limitations section in some pages, while I don't have anything on top of mind.

https://docs.delta.io/latest/delta-uniform.html#limitations